src/types/common/enums/common.enums.ts

```diff
@@ -6,3 +6,9 @@ export enum ProficiencyLevelEnums {
   Professional = 'Professional',
   Specialized = 'Specialized'
 }
+
+export enum Size {
```

Comment:
we decided that in enums we use Enum keyword in the end of the name

Reply:
okey, changed